mitxgraders/__init__.py

```diff
@@ -1,5 +1,5 @@
 """
-MITx Gradling Library
+MITx Grading Library
```

Comment:
Did we seriously have that typo? lol

Reply:
Yeah. I figure catching that one redeems me a bit.

Reply:
👍 I'm guessing that means you found a way to spell check in atom ;-)